spec.html

```diff
@@ -4206,6 +4224,28 @@ <h1>The ClassFieldDefinition Record Specification Type</h1>
               The initializer of the field, if any.
             </td>
           </tr>
+          <tr>
+            <td>
+              [[DecoratorInitializers]]
+            </td>
+            <td>
+              a list of ECMAScript function objects, or ~empty~
```

Comment:
```suggestion
              a List of ECMAScript function objects, or ~empty~
```
any reason not to default it to an empty List?